instrumentation/opentelemetry-instrumentation-celery/test-requirements-1.txt

```diff
@@ -1,6 +1,6 @@
 amqp==5.2.0
 asgiref==3.8.1
-billiard==4.2.0
+billiard==4.2.1
```

Comment:
Does this change something?

Reply:
Nope. They just released some fixes to work on py313 like: `Adapt for logging lock internal changes in Python3.13`